packages/fuel-gauge/test/fixtures/forc-projects/proxy-src14-contract/Forc.toml

```diff
@@ -0,0 +1,8 @@
+[project]
+authors = ["Fuel Labs <contact@fuel.sh>"]
+entry = "main.sw"
+license = "Apache-2.0"
+name = "proxy-src14-contract"
+
```

Comment:
Should this have the `proxy` flag?

Reply:
@danielbate Good catch.

Reply:
From my investigation, I actually think the flag goes on the target contract. Not the SRC14 compliant contract.